src/Dfc.CourseDirectory.Web/ViewComponents/Courses/ChooseVenue/Default.cshtml

```diff
@@ -1,38 +1,24 @@
-﻿@using Dfc.CourseDirectory.Models.Models.Courses;
-@using Dfc.CourseDirectory.Web.Helpers;
-@addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
+﻿@addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
```

Comment:
Changes to this component will impact both CopyCourseRun and EditCourseRun. It's worth noting that the functionality hasn't changed - most of this delta is a clean up of code and styling fixes, however the AddVenue link has changed its text value and moved from being abeam the dropdown menu to beneath it.